app/shortcuts/shortcuts.tsx

```diff
@@ -138,6 +142,12 @@ export class Shortcuts {
           if (!this.preferences.keyboardShortcutsEnabled) {
             return;
           }
+          // Don't run when typing into a text box.
+          let activeElement = document.activeElement;
+          if (activeElement.tagName === "INPUT" && activeElement.type === "text") {
```

Comment:
nit: can handle textarea here too (we use those e.g. in the secrets page)

```suggestion
          if ((activeElement.tagName === "INPUT" && activeElement.type === "text") || activeElement.tagName === "TEXTAREA") {
```